### ISSUE TITLE
Unable to respond to survey on YNMP

### ISSUE THREAD

**Issue by andylolz** [DemocracyClub/Website#12]
when clicking anything on the first page of the survey, the chrome javascript console says:
```
research.min.js:5 Uncaught TypeError: Cannot read property 'close' of undefined
```

**Comment by symroe** [comment]
This was caused by foundation messing with the version of jQuery in the research file.  Fixed now.

**Comment by andylolz** [comment]
Fixed in DemocracyClub/YourNextMP-Read@3b696998
